test/pools/integration/Cases.t.sol

```diff
@@ -219,4 +221,44 @@ contract TestCases is BaseTest {
         (uint128 totalIssuance2,) = multiShareClass.metrics(scId);
         assertEq(totalIssuance2, 55);
     }
+
+    function testNotifyPricePoolPerShare() public {
+        (PoolId poolId, ShareClassId scId) = testPoolCreation();
+        D18 sharePrice = d18(100, 1);
+        D18 poolPerEUR = d18(1, 3); // 3 EUR = 1 USD
+        D18 poolPerUSDC_C2 = d18(1, 1); // 1 USDC_C2 = 1 USD
+
+        (bytes[] memory cs, uint256 c) = (new bytes[](5), 0);
+        cs[c++] = abi.encodeWithSelector(hub.setTransientPrice.selector, EUR.addr(), poolPerEUR);
+        cs[c++] = abi.encodeWithSelector(hub.updatePricePoolPerShare.selector, scId, sharePrice, "");
+        // FIXME(wischli): Doesn't use transient price of first msg
```

Comment:
Just a highlighting this comment

Reply:
Yeah this whole test is a big question mark for me. Waiting for @mustermeiszer response [here](https://github.com/centrifuge/protocol-v3/pull/184#discussion_r2031618995).